blog/_posts/2016-11-29-intercontinental-gotober.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Intercontinental conference meeting @GOTOber in Berlin
+layout: post
+created_at: Tue Nov 29 2016
+permalink: blog/2016-11-29-intercontinental-gotober
+current: blog
+author: Marie & Thea, Taneea & Vrinda, Mayar & Nada
+twitter: LoadToCode
+categories:
+- student-posts
+- conferences
+---
+![RGSoC goes GOTOber](/img/blog/2016/20161128-GOTOber_1.gif)<font color="grey"><small><i>Team Twitches, Team Ruby's Secret and Team LoadToCode go GOTOber</i></small></font>
+
+So GOTO Berlin meant two days of conference in Berlin on November 14th & 15th, and it is all about learning, networking and meeting developers. But it was not just another conference. Not only Team LoadToCode from Berlin was attending, but also Team Twitches from New Delhi/India and Team Ruby's Secret from Cairo/Egypt \o/!
```

Comment:
maybe you could add links to all three teams' twitter accounts here in this paragraph, since you name all three? Would be awesome :)